### PR TITLE
Spectra v2.9.1

### PR DESCRIPTION
**Pull request by adi3890**
* Improvement: Container - Added a fallback image option for the video background type.
* Improvement: Container - Added a new 'Centralized Position' setting for custom image background positioning.
* Improvement: Global - Added the new 'X' logo for Twitter into the icon library.
* Fix: Container - Resolved an issue wherein the background video opacity would not work as intended.
* Fix: Container - Resolved an issue wherein the flex properties would not work as intended on the front-end when blocks were added inside a video-background Container.
* Fix: Buttons - Resolved an issue in which the margins added to the button group would get overridden by child button margins.
* Fix: FAQ - Resolved an issue in which the default content padding would appear differently in the editor and the front-end.
* Fix: Form - Added a note for the Form block regarding duplicate name fields requiring distinct labels.
* Fix: Icon List - Resolved an issue wherein the vertical alignment would not appear as intended in the editor's mobile view.
* Fix: Post Grid/Carousel/Timeline - Resolved an issue wherein a `preg_replace` warning could appear.
* Fix: Slider - Resolved an issue wherein the block would not work as intended in the Astra widget customizer.
* Fix: Table Of Contents - Resolved an issue wherein the Table Of Contents collapse feature would not work as intended.
* Fix: Global - Resolved an issue with Full Site Editing (FSE) in which changes applied to page templates would not be visible when File Generation was enabled.
* Fix: Global - Resolved an issue in which the Spectra blocks were unable to be previewed in the widget editor customizer.

### CODE REVIEW

**Review by rahulvarma722 — APPROVED** [review]

**Review by AbhijitNage123 — APPROVED** [review]

File: classes/class-uagb-post-assets.php [hunk 1]
```diff
@@ -1453,4 +1546,35 @@ public function merge_array_string_values( $array1, $array2 ) {
 
 		return array_merge( $array1, $array2 );
 	}
+
+	/**
+	 * Handle the block assets when blocks type will be core/pattern.
+	 *
+	 * @param array $block The block array.
+	 * @since X.X.X
```
Comment (bsf-zanev) [inline classes/class-uagb-post-assets.php:1554]: It appears that this did not get updated.

Comment (bsf-zanev) [reply classes/class-uagb-post-assets.php:1554]:
```suggestion
	 * @since 2.9.1
```

Comment (AbhijitNage123) [reply classes/class-uagb-post-assets.php:1554]: Done here - https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3592

Comment (adi3890) [reply classes/class-uagb-post-assets.php]: @bsf-zanev please close the PR if issue resolved.

**Pull request merged**